app/views/runs/_quick_stats.slim

```diff
@@ -0,0 +1,53 @@
+- previous_upload = @run.user.try(:previous_upload_for, @run.category)
+
+article data-turbolinks-temporary=true
+  .row
+    .col-sm-3
+      .statcard.p-3
+        h2.statcard-number
+          span> = format_ms(run.duration_ms(timing))
+          - if previous_upload.present?
+            - if run.duration_ms(timing) <= previous_upload.duration_ms(timing)
+              small.delta-indicator.delta-negative.text-success.tip-top title='Since last PB'
+                = format_ms_casual(previous_upload.duration_ms(timing) - run.duration_ms(timing))
+            - else
+              small.delta-indicator.delta-positive.text-danger.tip-top title='Since last PB'
+                = format_ms_casual(run.duration_ms(timing) - previous_upload.duration_ms(timing))
+        span.statcard-desc Time
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number
+          span> = format_ms(run.sum_of_best_ms(timing))
+          - if previous_upload.present?
+            small.delta-indicator.delta-negative.text-success.tip-top title='Since last PB'
+              = format_ms_casual(previous_upload.sum_of_best_ms(timing) - run.sum_of_best_ms(timing))
+        span.statcard-desc = 'Sum of best'
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number = format_ms(run.duration_ms(timing) - run.sum_of_best_ms(timing))
+        span.statcard-desc Potential timesave
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number = Run.program(run.timer)
+        span.statcard-desc = link_to('Timer', Run.program(run.timer).website, target: '_blank', class: 'text-muted')
+  .row
+    .col-sm-3
+      .statcard.p-3
+        h3.statcard-number
+          span> = run.attempts
+          - if previous_upload.present?
+            small.delta-indicator.delta-positive.tip-top title='Since last PB'
+              = run.attempts - previous_upload.attempts
```

Comment:
This number can be negative but still show a green arrow.

Reply:
Fixed